### PR TITLE
Added schema.json and the ajv-cli validator

### PR DESCRIPTION
**Pull request by spiray**
This PR includes the following things - 
1. Added schema.json
2. installed ajv-cli as a dev dependency (I had to install it globally to use the cli).
3. Added a validate script to the package.json
4. Added the validate script to the pre-commit script and the test script.

### CODE REVIEW

File: schema.json [hunk 1]
```diff
@@ -0,0 +1,100 @@
+{
+    "definitions": {},
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "http://example.com/root.json",
```
Comment (w3cj) [inline schema.json]: You can remove the $id line.

File: schema.json [hunk 2]
```diff
@@ -0,0 +1,100 @@
+{
+    "definitions": {},
```
Comment (w3cj) [inline schema.json]: You can remove the definitions.

File: schema.json [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "http://example.com/root.json",
+    "type": "array",
+    "title": "The Root Schema",
```
Comment (w3cj) [inline schema.json]: Lets give it a descriptive title like "Contributors to the Coding Garden Community App"

**Review by w3cj — CHANGES_REQUESTED** [review]
Looks great.  Only requested changes are to update the schema file to be a bit more descriptive.

Instead of all of the titles being "The _ Schema" give them a descriptive name like "Github Username", "3 Letter Country Code" etc.

**Comment by spiray** [comment]
Made all the requested changes @w3cj

File: schema.json [hunk 4]
```diff
@@ -51,7 +49,7 @@
             "countryCode": {
                 "$id": "#/items/properties/countryCode",
                 "type": "string",
-                "title": "The Countrycode Schema",
+                "title": "3 Charachter Country Code",
```
Comment (w3cj) [inline schema.json]: Not to nitpick... but Character is spelled wrong. Maybe call it 3 Letter Country Code? Don't think there are any country codes with numbers in them..

**Review by w3cj — CHANGES_REQUESTED** [review]
Update spelling / wording.

File: schema.json (hunk 4 above)
Comment (spiray) [reply schema.json]: Please nitpick! Updated in latest commit.

**Review by w3cj — APPROVED** [review]

**Pull request merged**